src/Altinn.Platform/Altinn.Platform.PDF/src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-server.port=5070
+server.port=5071
```

Comment:
Is this change intentional? If this is the case we have to update  helm chart and `docker-compose.yml` in local test to reference the new port.

Reply:
This is not intentional. Should not be commited!